environment-dev.yml

```diff
@@ -17,6 +17,11 @@ dependencies:
     - types-aiofiles
     - sqlalchemy-stubs
     # documentation
+    
+    # Temporary dependency for the new Panel Chat Interface.
+    # TODO: remove this dependency once the new Panel Chat Interface is released,
+    # and pin Panel's version
+    - git+https://github.com/holoviz/panel@chat_components
```

Comment:
I've added it to `pyproject.toml`. Release should be soon and until then we can `pip install --pre panel`.